src/Components/test/testassets/BasicTestApp/FormsTest/TypicalValidationComponent.razor

```diff
@@ -96,6 +104,13 @@
             </InputRadioGroup>
         </InputRadioGroup>
     </p>
+    <p class="radio-group-bool-values">
+        T/F: 7 * 3 = 21<br>
+        <InputRadioGroup @bind-Value="person.IsRadioMathStatementTrue">
+            <InputRadio Value="true" />true<br>
+            <InputRadio Value="false" />false<br>
```

Comment:
I'm guessing that if the code here was `Value="@true"` and `Value="@false"` it would still work the same, would it?

If I'm reading the code here correctly, `<InputRadio Value="true" />` will be an `InputRadio<string>` rather than an `InputRadio<bool>`. Does it make any difference if the developer makes it into an `InputRadio<bool>`?

I think it doesn't make any difference because the binding machinery is really on the `InputRadioGroup` which will be an `InputRadioGroup<bool>` in both cases.

Reply:
Since `Value` is a parameter of `InputRadio`, not an HTML attribute, isn't 'true' interpreted as the value `true` rather than the string "true"? In other words, I think `true` and `@true` are equivalent here (correct me if I misinterpreted your comment).

That said, the `<InputSelect>` case is a little weird in exactly this regard:
```razor
<InputSelect @bind-Value="ignoreTheTypeOfThis">
    <!-- We can set the value attribute to an enum, because the bind formatter properly serializes it -->
    <option value="@SomeEnum.Choice">Choice</option> <!-- Good -->

    <!-- But we can't do this with a bool, because now 'value' is treated as a conditional attribute -->
    <option value="@true">true</option> <!-- Bad! -->

    <!-- So, we have to use a string -->
    <option value="true">true</option> <!-- Good? -->
</InputSelect>
```